### PR TITLE
MCA setup improvements

### PR DESCRIPTION
**Pull request by smathangi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ACA-60

### Change description ###

Automated as many steps as possible and documented in READMEs to ensure setup and running of MCA feature tests is as simple as possible

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: aca-docker/bin/set-environment-variables.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    file=$1
+    if [ -f ${file} ]
+    then
+        osName="$(uname -s)"
+        echo "Setting env variables from [$file] on [$osName]."
+        while IFS="=" read -r key value
+        do
+            if [[ "Darwin" == "$osName" ]];then
+                command="export $key=$value"
+                $command
```
Comment (mattnayler) [inline aca-docker/bin/set-environment-variables.sh:19]: If this fails on EVs that have spaces in values : would switching to `source ./aca-docker/bin/env_variables_all.txt ` work either in this script or as an alternative to it?  (if so update script and/or README)

**Review by mattnayler — CHANGES_REQUESTED** [review]
Works for me locally: 👍  but see minor changes in comments.

**Review by mattnayler — APPROVED** [review]
👍  LGTM: under assumption that switch to an update `befta_fw` which resolves AAT callback issues is also included.

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**